compiler/z/codegen/S390GenerateInstructions.cpp

```diff
@@ -3380,3 +3240,74 @@ TR::Instruction *generateZeroVector(TR::Node *node, TR::CodeGenerator *cg, TR::R
    {
    return generateVRIaInstruction(cg, TR::InstOpCode::VREPI, node, vecZeroReg, 0, 3);
    }
+
+#ifdef J9_PROJECT_SPECIFIC
+
+/**
+ * \brief
+ *    Determines if an instruction can throw a decimal overflow exceptions.
+ *
+ * \param op
+ *    The instruction to check.
+ *
+ * \return
+ *    true if \p op is designed to throw decimal overflow exception according to the PoP; false otherwise.
+ *
```

Comment:
The name is slightly confusing with both `is` and `Can` in the name. Would `canThrowDecimalOverflowException` be better?